src/validation.js

```diff
@@ -59,18 +64,50 @@ export function validateField(fieldUri, options) {
   );
 }
 
+/**
+ * Returns a list of Statements for each validation in the form graph that is attached to the given field
+ *
+ * @param {NamedNode} fieldUri
+ * @param {{store: ForkingStore, formGraph: NamedNode, severity?: NamedNode}} options
+ * @returns {Statement[]}
+ */
 export function validationsForField(fieldUri, options) {
-  const { store, formGraph } = options;
-  const v2Result = store.match(
+  const { store, formGraph, severity } = options;
+
+  /** @type {Statement[]} */
+  let validations = store.match(
     fieldUri,
     FORM("validatedBy"),
     undefined,
     formGraph
   );
-  if (v2Result.length > 0) {
-    return v2Result;
+
+  if (validations.length === 0) {
+    // v1 model fallback, for backwards compatibility
+    validations = store.match(
+      fieldUri,
+      FORM("validations"),
+      undefined,
+      formGraph
+    );
   }
-  return store.match(fieldUri, FORM("validations"), undefined, formGraph);
+
+  validations = validations.filter((statement) => {
+    const constraint = statement.object;
+    const severityType = store
+      .match(constraint, SHACL("severity"), null, formGraph)
+      .at(0)?.object;
+
+    const SH_VIOLATION = SHACL("Violation");
```

Comment:
Do we want to export the different severity options as nodes from this addon? At the moment, consumers will have to create the `SHACL("Warning")` nodes themselves if they want to pass it in as an option.

Reply:
i for one would be grateful for such an export. I do understand that it's a bit of a luxury and extra maintenance for this repo, but i don't think these levels are likely to change?

Reply:
Yea, I wasn't sure where to draw the line exactly. How would you name the exports, and from where would you export them?

An export for each level, grouped in an object, ..?

At the moment the package only allows importing from the index.js file, maybe we should expose a `/validation` import?



1. `import { Warning, Violation } from '@lblod/submission-form-helpers`;
2. `import { Warning, Violation } from '@lblod/submission-form-helpers/validation`;
3. `import { Severity } from '@lblod/submission-form-helpers'; Severity.Warning;`